src/services/bulksubmitstatus.service.ts

```diff
@@ -29,17 +29,16 @@ export async function bulkSubmitStatus(req: any, res: any) {
 
   const clientId = `${submitter.value}-${submissionId}`;
 
-  logger.debug(`Retrieving bulkStatus entry for client: ${clientId}`);
+  logger.debug(`Retrieving submission status entry for client: ${clientId}`);
+  const submissionStatus = getBulkSubmissionStatus(submitter.value, submissionId);
 
-  const bulkStatus = getBulkImportStatus(clientId);
-
-  if (!bulkStatus) {
+  if (!submissionStatus) {
     logger.debug(`Writing unable to find bulk import request OperationOutcome to file for client: ${clientId}`);
 
-    throw new NotFoundError(`Could not find bulk import request with id: ${clientId}`);
+    throw new NotFoundError(`Could not find submission request with id: ${clientId}`);
```

Comment:
```suggestion
    throw new NotFoundError(`Could not find submission status entry with id: ${clientId}`);
```
I know the previous version wasn't very consistent, but maybe this could be good for consistency?